Service-Broker-Setup: Use values files instead of command line

mook-as opened the pull request:
This looks more like the SCF deployment mechanism, but there is less
interpolation and more manual editing.

mook-as committed:
Service-Broker-Setup: Use values files instead of command line

This looks more like the SCF deployment mechanism, but there is less
interpolation and more manual editing.

andreas-kupries commented on Service-Broker-Setup.md:
````diff
@@ -13,50 +13,66 @@ a CAP cluster.
+```yaml
+env:
+  # Database access credentials; the given user must have privileges to create
+  # delete both databases and users
````
...create __and__ delete... . The `and` is missing in the sentence.

mook-as replied:
Fixed in 9bdb11490e74fdbd3a616b863807e1bb7934077b

mook-as committed:
Service-Broker-Setup: fix grammar

Also add warning about the future-looking state of the document, and do
some line wrapping.

troytop commented on Service-Broker-Setup.md:
```diff
@@ -13,50 +16,69 @@ a CAP cluster.
+
+  SERVICE_TYPE: mysql # Optional
+
+# The whole "kube" section is optional
```
Omit

troytop commented on Service-Broker-Setup.md:
```diff
@@ -13,50 +16,69 @@ a CAP cluster.
-    | `CF_CA_CERT` | The certificate for the internal certificate authority for the CAP deployment | `CF_CA_CERT="$(kubectl get secret --namespace $NAMESPACE secret-$REVISION -o jsonpath='{.data.internal-ca-cert}' \| base64 -d)"`
-    | `UAA_CA_CERT` | The certificate for the internal certificate authority for the UAA deployment | `UAA_CA_CERT="$(kubectl get secret --namespace $NAMESPACE secret-$REVISION -o jsonpath='{.data.uaa-ca-cert}' \| base64 -d)"`
-    | `SIDECAR_NAMESPACE` | The Kubernetes to install the sidecar to; may be the same as the CAP namespace | `SIDECAR_NAMESPACE=${NAMESPACE}`
+When deploying the chart, a Kubernetes namespace to install the sidecar to is
```
What is a sidecar (as far as the user knows)?

troytop reviewed (approved):
Some small things could be changed, but that can happen later. The process is workable with these steps.
